build/script.js

```diff
@@ -136,19 +136,19 @@ function createCompileTask(translation) {
  */
 function compileES6(translation) {
   let externs = [
-    path.join(conf.paths.nodeModules, 'google-closure-compiler/contrib/externs/angular-1.5.js'),
+    path.join(conf.paths.nodeModules, 'google-closure-compiler/contrib/externs/angular-1.6.js'),
```

Comment:
Not really, I have just noticed that this wasn't updated. Since it's small I haven't considered moving it into separate PR.